creator-node/src/app.ts

```diff
@@ -175,11 +173,40 @@ export const initializeApp = (port: number, serviceRegistry: any) => {
             }
           }
         } catch (e: any) {
-          reqWithLogger.logger.warn(
-            `DurationTracking || Could not match on regex: ${e.message}`
+          req.logger.warn(
+            { middleware: 'PrometheusMiddleware', function: 'normalizePath' },
+            `Could not match on regex: ${e.message}`
           )
         }
         return path
+      },
+      // Function taking express req as an argument and determines whether the given request should be excluded in the metrics
+      // Technically, this function is redundant because we specify the routes to match on in the first
+      // param of app.use([paths,]). This fn is used for specific metric tracking
+      bypass: function (_req) {
```

Comment:
this middleware is meant to be passed into all routes. but in line 148, i specified that only routes that exist should have this middleware https://github.com/AudiusProject/audius-protocol/pull/4011/files/247982914c5ad372a8daff34af6e1a224e90108e#diff-c2c395f7542318c38d88b2c6a3ce4646927f81b754362a8dc75ffdedb68d047dL148

this fact renders the `bypass` fn redundant. however, for the sake of adding additional metric tracking, im taking over the `bypass` function (which is why we always return `false` because we never want to bypass)